pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+Hi there!
+Thanks for contributing to Swarm!
+
+Before opening a Pull Request or reviewing a Pull Request, please get yourself acquainted with the following guidelines:
+
+1. Open PRs preemptively (i.e. whenever you start working on it), especially for the case of large tasks
+2. Create PRs with a healthy amount of description. A PR without a description will not be approved
+3. PR names should adhere to the following standard: "(affected package(s)): description". examples: "storage: fix panic on hasherstore put", "network, storage, p2p: integrate capabilities api". if many (many) packages are affected, use "all" instead of individual names
+3. Mark the PR with a `ready for review` label once it is. If a second(or more) round of reviews is required, remove the `ready for review` label and add a `ready for another review` label
```

Comment:
use "in progress" inbetween

Reply:
TBH i find the `in progress` label redundant and never use it. if the package is not marked ready for review it means it is in progress and we should not review it. not sure why we need a label for every permutation of the universe

Reply:
If it is first "ready for review" and then needs some more work I change it to "in progress" while it's not ready for review anymore. I think this protocol makes it more clear when to re-review PRs.

Reply:
as @nolash says it is in order for a second reviewer to judge if submitter works on fixes atm.